sdk/iot_deviceupdate/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "azure_iot_deviceupdate"
-version = "0.1.0"
+version = "0.12.0"
```

Comment:
It appears that `azure_iot_deviceupdate` has not previously been published/updated.
I see that there was an earlier PR to disable publishing: https://github.com/Azure/azure-sdk-for-rust/pull/1138
Have you made a subsequent change to re-enable updating and/or publishing (if it is now ok to be published)?

Reply:
This crate is still not ready to be published, but as we standardized on a single version across all of the crates, this has no negative impact when we move forward to release this crate eventually.